lib/src/pdfjs_global.dart

```diff
@@ -29,113 +29,159 @@ enum VERBOSITY_LEVELS {
 }
 
 class PDFJS {
-  static final JsObject _pdfjsContext = context['pdfjsLib'] as JsObject;
+  /// the local copy of versionSafePdfJsContext once initialized
+  static JsObject _pdfJsContext;
+
+  /// the local copy of versionSafePdfJsViewerContext once initialized
+  static JsObject _pdfJsViewerContext;
+
+  /// Getter for obtaining the pdfjs context for various versions
+  /// of pdf.js
+  static JsObject get versionSafePdfJsContext {
+    if(_pdfJsContext != null) {
+      return _pdfJsContext;
+    }
+    
+    // ^v2.5.207
+    if(context['pdfjsLib'] != null) {
+      _pdfJsContext = context['pdfjsLib'] as JsObject;
+      return _pdfJsContext;
+    }
+    // v1.10.90+4
+    else if(context['PDFJS'] != null) {
+      _pdfJsContext = context['PDFJS'] as JsObject;
+      return _pdfJsContext;
+    }
+
+    return null;    
+  }
+
+  /// Getter for obtaining the pdfjs viewer context for various versions
+  /// of pdf.js
+  static JsObject get versionSafePdfJsViewerContext {
+    if(_pdfJsViewerContext != null) {
+      return _pdfJsViewerContext;
+    }
+
+    // ^v2.5.207
+    if(context['pdfjsViewer'] != null) {
```

Comment:
This is needed, as can be seen [here](https://github.com/Workiva/pdfjs_dart/pull/34/files?file-filters%5B%5D=.dart#diff-125e4040cedf5dd1bcdf52505b24c80c62e23709acc20e5cd2f477885b436df9L41).

v1.10.90.4 of the interop looks for `context['PDFJS']`, whereas newer versions look for `context['pdfjsViewer']`